core/src/main/scala/kafka/server/KafkaApis.scala

```diff
@@ -3661,6 +3662,13 @@ class KafkaApis(val requestChannel: RequestChannel,
     }
   }
 
+  def handleConsumerGroupDescribe(request: RequestChannel.Request): CompletableFuture[Unit] = {
+    // TODO: This method is Work-In-Progress
+    // It will be finished in the second PR of KAFKA-14509
```

Comment:
nit: We usually try to avoid leaving TODOs in the code. I think that we can remove it as we know that we will do this in the second part of this JIRA.